scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -176,7 +177,57 @@
       $('table').stickyTableHeaders({
         fixedOffset: $(".navbar-fixed-top")
       });
-    })
+    });
 
   </script>
 {% endblock %}
+
+{% macro str_status(variant) %}
+<a class="badge badge-secondary text-white"
+  data-toggle="popover"
+  data-html="true"
+  data-trigger="hover click"
+  data-content="{% if variant.str_normal_max %}
+    <table>
+      <tr><td>Reference size</td><td>{{ variant.str_normal_max }}</td></tr>
+      <tr><td>Normal max</td><td>{{ variant.str_normal_max }}</td></tr>
+      <tr><td>Pathologic min</td><td>{{ variant.str_pathologic_min }}</td><tr>
+      <tr><td colspan=2>&nbsp;</td></tr>
+      {% if variant.str_swegen_mean %}
+        <tr><td>SweGen Z-score</td><td>
+          {{ ((variant.alternative|replace("STR", "")|replace("<", "")|replace(">", "")|int - variant.str_swegen_mean ) / variant.str_swegen_std) | round(2) }}
+        </td></tr>
+        <tr><td>SweGen mean</td><td>{{variant.str_swegen_mean|round(2)}}</td></tr>
+        <tr><td>SweGen std</td><td>{{variant.str_swegen_std|round(2)}}</td></tr>
+      {% endif %}
+    </table>
+    {% endif %}
+    {% if not variant.str_normal_max %}
+      <div>Normal range not provided. Consider updating to a new version of Stranger.</div>
+    {% endif %}"
+  title="">{{ variant.str_status }}</a>
+{% endmacro %}
+
+{% macro str_locus_info(variant) %}
+  <a
+  href="{{ variant.str_source_link }}"
+  class="link text-black"
+  target="_blank"
+  data-toggle="popover"
+  data-html="true"
+  data-trigger="hover"
+  data-content="
+  <div class='text-left'>
+    {% if variant.str_disease %}
+      <div><strong>Disease:</strong> {{ variant.str_disease }} ({{ variant.str_inheritance_mode }})</div>
+    {% else %}
+      <div>Disease information not provided with locus. Consider updating to a new version of Stranger.</div>
```

Comment:
Doesn't break the demo case at least:

![image](https://user-images.githubusercontent.com/28093618/106733627-697a4f80-6612-11eb-97e4-a1a820e9dc5f.png)

Reply:
I suppose that will have to be the sign to the users for now that filters are not working for this case. As soon as it is updated in MIP we can also let ppl know from what version/date etc.